src/components/BottomBar/index.js

```diff
@@ -38,12 +33,11 @@ export default class BottomBar extends Component {
 
   render() {
     const { classes } = this.props;
-    const { syncBlockNum, syncBlockTime } = this.props.store.global;
-
+    const { syncBlockTime, syncBlockNum, peerNodeCount } = this.props.store.global || { syncBlockTime: 1, syncBlockNum: 1, peerNodeCount: 1 };
```

Comment:
lets not set the defaults here if they are undefined. the store defaults should be sufficient enough.

Reply:
good point